content.js

```diff
@@ -0,0 +1,13 @@
+function checkForOnionMeta() {
```

Comment:
I can't remember how content scripts work. Does this define a global function for all browser pages?

Reply:
I can't tell ya with 100% certainty, still wrapping my head around how browser extensions work, but I believe so. But I believe the scope is limited to the extension itself.